source/_integrations/nest.markdown

```diff
@@ -202,6 +202,8 @@ Once your developer account is set up and you have a valid `nest` entry in `conf
 
 - *Subscriber error: Subscription misconfigured. Expected topic name to match ...* means that the topic name in the Google Cloud Console was entered incorrectly. The topic name comes from the Device Console and must start with `projects/sdm-prod/topics/`. It is easy to make the mistake of creating a new topic rather than manually entering the right topic name.
 
+- *Error 400: Invalid Request* This error typically happens when there is a mismatch with how you are accessing Home Assistant and the Authorized Redirect URI you entered for OAUTH. If you used a publicly facing URL make sure that you are accessing Home Assistant with the same URL when you setup the Integration. You will definitely hit this error if you are accessing Home Assistant by local IP during intial setup.
```

Comment:
Since this is URL related, perhaps this should go up next to the Error 500 above?

Reply:
Done, thank you :)